crates/language/src/language_settings.rs

```diff
@@ -95,6 +95,8 @@ pub struct LanguageSettings {
     pub use_autoclose: bool,
     /// Which code actions to run on save
     pub code_actions_on_format: HashMap<String, bool>,
+    // Controls how the editor handles the autoclosed characters.
+    pub always_handle_autoclosed_character: bool,
```

Comment:
I am not 100% sure this is required.
I've followed how the `use_autoclose` is handled but I might have over-engineered it and controlling it globally would be more than enough.

Reply:
I think it's fine to allow overwriting these per language.